homeassistant/components/mqtt/climate.py

```diff
@@ -640,7 +640,7 @@ def preset_mode(self):
             return self._hold
         if self._away:
             return PRESET_AWAY
-        return None
+        return PRESET_NONE
```

Comment:
@MartinHjelmare Do you think this is OK, or should we return `None` until we have received away or old mode, or set a preset in optimistic mode? Maybe that just adds unnecessary complexity?

Reply:
I think it's ok like this since it will only be used for entities that support presets. There's no command that actively results in `PRESET_NONE` so I think it's ok to default it.

Reply:
Great, then this is OK to merge?